workspaces/js-x-ray/src/types/estree.ts

```diff
@@ -5,6 +5,10 @@ export type Literal<T> = ESTree.Literal & {
   value: T;
 };
 
+export type TemplateLiteral<T> = ESTree.TemplateLiteral & {
+  value: T;
+};
```

Comment:
Hmm this seems wrong to me, there no value in TemplateLiteral

Reply:
yeah I just copy paste too fast .hahah